llpc/test/shaderdb/partial_pipeline_cache_test/partial_pipeline_cache_test.ll

```diff
@@ -0,0 +1,19 @@
+; Test that,
+;   a.the partial pipeline cache works as expected.
+;     If the partial pipeline cache is enabled the new pipeline will first check and use the cached data instead of compiling.
+;   b.rodata section is merged correctly.
+;     If the .rodata section comes from a cached partial pipeline, there will be a .cached string appended to the original section name.
+; The test sequence is,
+;   a.	Build 3 pipelines: P1(Vs1, Fs1), P2(Vs1, Fs2), P3(Vs2, Fs1).
+;   b.	Give all 3 pipelines to amdllpc with shader cache enabled, and the stage access will be,
+;           miss, miss, hit, miss, miss, hit
```

Comment:
This test description is excellent, it's very clear what is being tested. Thanks!